src/components/global-styles.tsx

```diff
@@ -53,6 +53,12 @@ const SizeStyles = css`
       .onboarding-text {
         text-align: center;
       }
+
+      .unlock-text {
+        padding-left: 16px;
+        padding-right: 16px;
+        text-align: center;
+      }
```

Comment:
Couldn't this go the component?

Reply:
It can't (that I know of) bc it changes depending on if it's full page or in the popup so it is used here in the mode styling, that is how it is currently setup but I think we talked about changing it when we do the new onboarding. It is only centered in the full page mode and left aligned in the popup.

Reply:
```
type ViewMode = 'full' | 'popup';
function getViewMode(): ViewMode {
  return document.location.pathname.startsWith('/index.html') ? 'full' : 'popup';
}
```

Reply:
Ok, yep, I can add that to this PR. Am I supposed to refactor all uses of the mode styling here too? I was thinking we would tackle it with new onboarding designs but happy to address with this work.

Reply:
That's not necessary, but you can if you like. Leaving to onboarding is also fine, main thing is not adding to the existing styles.